src/rest-server/src/models/v2/job/runtime-env.js

```diff
@@ -43,6 +43,7 @@ const generateFrameworkEnv = (frameworkName, config) => {
     env[`PAI_MIN_SUCCEEDED_TASK_COUNT_${taskRole}`] =
       (tasks.completion && 'minSucceededInstances' in tasks.completion && tasks.completion.minSucceededInstances) ?
       tasks.completion.minSucceededInstances : (tasks.instances || 1);
+    env['PAI_CURRENT_TASK_ROLE_NAME'] = taskRole;
```

Comment:
what's the purpose of this line

Reply:
Before this change, the `PAI_CURRENT_TASK_ROLE_NAME` refer to the `metadata.annotations['FC_TASKROLE_NAME']`. And it can not contain characters such as `_`. It's not consistent with real task role name.